regression/tests/helpers.py

```diff
@@ -0,0 +1,47 @@
+"""
+Test helper functions.
+"""
+import os
+
+
+def visit_all(pages):
+    """
+    Visit each page object in `pages` (an iterable).
+    """
+    for page in pages:
+        print "Visiting: {}".format(page)
+        page.visit()
+
+
+class LoginHelper():
```

Comment:
This should really be `LoginHelper(object)`. We can get to that when we look at pylint.